idp-service/src/test/java/io/mosip/idp/flows/AuthCodeFlowTest.java

```diff
@@ -171,9 +172,9 @@ private PrivateKey getRelyingPartyPrivateKey(String relyingPartyId) throws Excep
 
     private String getUserInfo(String accessToken) throws Exception {
         MvcResult result = mockMvc.perform(MockMvcRequestBuilders.get("/oidc/userinfo")
-                        .accept(MediaType.APPLICATION_JSON_VALUE)
+                        .accept("application/jwt")
```

Comment:
I remember we need a similar change in AuthorizationCodeFlowParameterizedTest.java also, those test cases are not failing ?

Reply:
Test cases are not failing